src/components/helper/uc.helper.js

```diff
@@ -5,4 +5,12 @@ const validateObjectId = (objectId) => {
 	if (!isValidObjectId(objectId)) throw new ValidationError('a valid objectId is required', { objectId });
 };
 
-module.exports = { validateObjectId };
+const validateEmail = (email) => {
+	// eslint-disable-next-line no-useless-escape
+	const re = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
i think this expression should already exist and be reused.

Reply:
It is exists in the browser script only. I think it is a good place to have it here and reuse it for the future.

Reply:
not sure if it is the same string but it exists at least here: https://github.com/hpi-schul-cloud/schulcloud-server/blob/732aeb8cf654c6a6e8305dd7e893e2acb9d84fcf/src/utils/constants.js#L4